test/unit/pages/patientdata.test.js

```diff
@@ -3448,31 +3648,33 @@ describe('PatientData', function () {
     it('should set the `chartType` state to `daily`', () => {
       const wrapper = shallow(<PatientData.WrappedComponent {...defaultProps} />);
       const instance = wrapper.instance();
-      instance.dataUtil = new DataUtilStub();
 
       wrapper.setState({chartType: 'basics'});
 
       instance.handleSwitchToDaily();
       expect(wrapper.state('chartType')).to.equal('daily');
     });
 
-    it('should set `dataUtil._chartPrefs` to the `daily.chartPrefs` state', () => {
+    it('should call `getMostRecentDatumTimeByChartType`, `getChartEndpoints`, and then call `updateChart` with appropriate args', () => {
       const wrapper = shallow(<PatientData.WrappedComponent {...defaultProps} />);
       const instance = wrapper.instance();
+      wrapper.setState({timePrefs: { timezoneAware: true, timezoneName: 'utc' } })
 
-      const chartPrefs = { daily: 'daily prefs' };
-
-      wrapper.setState({ chartPrefs })
-      instance.dataUtil = new DataUtilStub();
+      instance.updateChart = sinon.stub();
+      instance.getMostRecentDatumTimeByChartType = sinon.stub().returns('2019-11-27T12:00:00.000Z');
+      instance.getChartEndpoints = sinon.stub().returns('endpoints stub');
 
       instance.handleSwitchToDaily();
-      expect(instance.dataUtil._chartPrefs).to.equal('daily prefs');
+      console.log(instance.getChartEndpoints.getCall(0).args)
```

Comment:
Console log left in? (there's a couple more lower down as well)

Reply:
Removing...